pkg/cliplugins/workspace/cmd/cmd.go

```diff
@@ -134,7 +134,7 @@ func New(streams genericclioptions.IOStreams) (*cobra.Command, error) {
 
 	listCmd := &cobra.Command{
 		Use:          "list",
-		Short:        "Returns the list of the personal workspaces of the user",
+		Short:        "Returns the list of the workspaces of the user",
 		Example:      "kcp workspace list",
```

Comment:
this should be deleted. It is already deprecated.

Reply:
Done in commit 9082614f6ca3142c0a4ea27d66ea017677190650